src/Categories/Monad/Construction/Kleisli.agda

```diff
@@ -0,0 +1,116 @@
+module Categories.Monad.Construction.Kleisli where
+
+-- Definition of kleisli triple as a relative monad
+-- and the equivalence of kleisli triple and monad
+-- see https://ncatlab.org/nlab/show/extension+system (the naming differs)
+
+open import Level
+open import Categories.Category
```

Comment:
Can you please use `Categories.Category.Core` (unless you really need more) and also `using` clauses and and for all `open import` below.

Reply:
Sure, fixed it!